### DISCUSSION TITLE
how to bump version and tag?

### DISCUSSION THREAD

**Discussion by AtticusZeller** [orhun/git-cliff#935] [Q&A] [answered]
it seems like bump only work for changelog ,nothing to do with git tag

**Comment by alerque** [comment, marked as answer]
Having worked (extensively) with changlog tooling that *does* trigger commit and tag operations, I really don't think I'd suggest `git-cliff` heads that way. It's much nicer just doing one job and not trying to handle the whole release process.

**Reply by AtticusZeller** [reply]
Thank you for your insight. ☺️I agree that it's often better for tools to focus on doing one job well rather than trying to handle the entire release process. 

so I'm curious about the manual workflow when the tagging is left to the user？
 thank you very much.

**Reply by alerque** [reply]
It depends on the project and project choices. Generating a changelog is *one step* of the release process for many projects, but prior to tagging there are quite often other chores that need doing. Typically the version number might get bumped inside some content files, e.g. `Cargo.toml` and `Cargo.lock` for Rust projects, `package.json` for JS projects, `configure.ac` for Autotools based C projects, and so on nearly to the stars and back. There might also be documentation fragments to update. Lua based projects are especially obnoxious requiring the release tag information to be in the filename of a newly minted rockspec file, so that has to be done prior or in conjunction with any release commit/tag operations.

There is tooling out there to automate this, I just don't think that the changelog generator should be the one piece of the puzzle that over-reaches and tries to handle the whole picture. I'd rather use `git-cliff` as one tool inside a release automation workflow.

**Reply by AtticusZeller** [reply]
> the changelog generator should be the one piece of the puzzle that over-reaches and tries to handle the whole picture


 i love this one 💓

**Reply by AtticusZeller** [reply]
OMG, I just found you are author of _Libertinus Fonts_

that's the front I applied in my current paper 📃 .

it's definitely gorgeous,thank you so much
